packages/simple-form/src/components/Form/Form.spec.js

```diff
@@ -3,12 +3,18 @@ import {Field} from 'redux-form'
 import {Provider} from 'react-redux'
 import {createStore} from 'redux'
 import {IntlStub, intlEnzyme} from 'tocco-test-util'
+import {ThemeProvider} from 'styled-components'
 
 import Form from './Form'
 
-const EMPTY_FUNC = () => {
+const theme = {
```

Comment:
maye consider moving a test theme to test-utils package

Reply:
I will open a separate pull request to create a centralized theme provider for tests in package tocco-test-util.